build.js

```diff
@@ -565,70 +646,15 @@ const buildIDLTests = (ast) => {
     });
 
     const members = flattenMembers(iface);
-
-    // Avoid generating duplicate tests for operations.
-    const handledMemberNames = new Set();
-
-    for (const member of members) {
-      if (handledMemberNames.has(member.name)) {
-        continue;
-      }
-
-      const isStatic =
-        member.special === 'static' || iface.type === 'namespace';
-
-      let expr;
-      const customTestMember = getCustomTestAPI(
-        iface.name,
-        member.name,
-        isStatic ? 'static' : member.type
-      );
-
-      if (customTestMember) {
-        expr = customTestMember;
-      } else {
-        switch (member.type) {
-          case 'attribute':
-          case 'operation':
-          case 'field':
-            if (isGlobal) {
-              expr = {property: member.name, owner: 'self'};
-            } else if (isStatic) {
-              expr = {property: member.name, owner: iface.name};
-            } else {
-              expr = {
-                property: member.name,
-                owner: `${iface.name}.prototype`,
-                inherit: member.special === 'inherit'
-              };
-            }
-            break;
-          case 'const':
-            if (isGlobal) {
-              expr = {property: member.name, owner: 'self'};
-            } else {
-              expr = {property: member.name, owner: iface.name};
-            }
-            break;
-          case 'constructor':
-            expr = {property: `constructor.${member.name}`, owner: iface.name};
-            break;
-          case 'symbol':
-            // eslint-disable-next-line no-case-declarations
-            const symbol = member.name.replace('@@', '');
-            expr = {property: `Symbol.${symbol}`, owner: `${iface.name}`};
-            break;
-        }
-      }
-
-      tests[`api.${iface.name}.${member.name}`] = compileTest({
-        raw: {
-          code: expr
-        },
-        exposure: Array.from(exposureSet),
-        resources: resources
-      });
-      handledMemberNames.add(member.name);
+    const memberTests = buildIDLChildrenTests(
+      members,
+      iface,
+      exposureSet,
+      isGlobal,
+      resources
+    );
+    for (const [k, v] of Object.entries(memberTests)) {
+      tests[`api.${iface.name}.${k}`] = v;
```

Comment:
If the BCD path is instead constructed in `buildIDLMemberTests`, then this can probably be removed, and custom tests can use the BCD paths too.